### ISSUE TITLE
explanations in video

### ISSUE THREAD

**Issue by mlefrancois19** [olidenis/Rocket_Elevators_Information_System#67]
mostly talks about admin panel ... on the database week, I was expecting a more information about databases

**Comment by olidenis** [comment]
Yeah we though that we could show everything with the admin panel instead of using Workbench and PgAdmin 4.
